apps/condo/domains/user/access/UserExternalIdentity.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Generated by `createschema user.UserExternalIdentity`
+ */
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+
+async function canReadUserExternalIntegrations ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    if (user.type === RESIDENT) {
```

Comment:
It looks like a common pattern for all user types

Reply:
Access model rewrited